src/order/filter/ultrafilter.lean

```diff
@@ -191,7 +191,7 @@ def ultrafilter (α : Type u) : Type u := {f : filter α // is_ultrafilter f}
 def ultrafilter.map (m : α → β) (u : ultrafilter α) : ultrafilter β :=
 ⟨u.val.map m, ultrafilter_map u.property⟩
 
-/-- The principal ultra-filter associated to a point `x`. -/
+/-- The 𝓟 ultra-filter associated to a point `x`. -/
```

Comment:
```suggestion
/-- The principal ultra-filter associated to a point `x`. -/
```
I think it'd be better not to use the notation in the doc string, since we can't easily "go to definition".